### DISCUSSION TITLE
about 08_Hand_Gesture_Recognition

### DISCUSSION THREAD

**Discussion by a5111295** [Ignitarium-Renesas/RZV2L_AiLibrary#10] [General]
when we use this application, 

 in the case of "if(hand_count == 0 && thresh_score > 70)",
this message is shown ,
" with low confidence"

what does it mean?

**Comment by a5111295** [comment]
I am sorry that it is "12-Hand Gesture Recognition_v2" not 08_Hand_Gesture_Recognition

**Reply by a5111295** [reply]
I moved to this comment to "issues #12"

**Reply by mohitb-ign** [reply]
Answered in issues section. Please have a look.
